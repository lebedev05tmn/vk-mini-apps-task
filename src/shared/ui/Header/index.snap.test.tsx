import { Header } from "./index";

const ReactTestRenderer = require("react-test-renderer");

describe("Headers render", () => {
  test("Should Header renders is Main", () => {
    const tree = ReactTestRenderer.create(
      <Header
        children={"Hello"}
        isMain={true}
        onBackClick={() => console.log("Hello word")}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("Should Header renders is not Main`", () => {
    const tree = ReactTestRenderer.create(
      <Header
        children={"Hello"}
        isMain={true}
        onBackClick={() => console.log("Hello word")}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
