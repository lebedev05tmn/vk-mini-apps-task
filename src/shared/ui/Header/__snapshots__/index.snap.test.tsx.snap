// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headers render Should Header renders is Main 1`] = `
<div
  className="vkuiPanelHeader vkuiInternalPanelHeader vkuiPanelHeader--android vkuiPanelHeader--static vkuiInternalPanelHeader--static vkuiPanelHeader--sep vkuiInternalPanelHeader--sep vkuiPanelHeader--no-before vkuiInternalPanelHeader--no-before vkuiPanelHeader--no-after vkuiPanelHeader--fixed vkuiPanelHeader--sizeX-none vkuiPanelHeader--sizeY-none"
>
  <div
    className="vkuiFixedLayout vkuiFixedLayout--vertical-top vkuiPanelHeader__fixed vkuiInternalPanelHeader__fixed"
    data-onboarding-tooltip-container="fixed"
    style={
      {
        "width": undefined,
      }
    }
  >
    <div
      className="vkuiPanelHeader__in"
      data-onboarding-tooltip-container="fixed"
    >
      <div
        className="vkuiPanelHeader__before vkuiInternalPanelHeader__before"
      />
      <div
        className="vkuiPanelHeader__content"
      >
        <span
          className="vkuiPanelHeader__content-in"
        >
          Hello
        </span>
      </div>
      <div
        className="vkuiPanelHeader__after vkuiInternalPanelHeader__after"
      />
    </div>
  </div>
  <div
    className="vkuiSeparator vkuiSeparator--padded vkui-sizeX--compact-mq"
  >
    <hr
      className="vkuiSeparator__in"
    />
  </div>
  <div
    className="vkuiSpacing vkui-sizeX--regular-mq"
    style={
      {
        "height": 16,
        "padding": "8px 0",
      }
    }
  />
</div>
`;

exports[`Headers render Should Header renders is not Main\` 1`] = `
<div
  className="vkuiPanelHeader vkuiInternalPanelHeader vkuiPanelHeader--android vkuiPanelHeader--static vkuiInternalPanelHeader--static vkuiPanelHeader--sep vkuiInternalPanelHeader--sep vkuiPanelHeader--no-before vkuiInternalPanelHeader--no-before vkuiPanelHeader--no-after vkuiPanelHeader--fixed vkuiPanelHeader--sizeX-none vkuiPanelHeader--sizeY-none"
>
  <div
    className="vkuiFixedLayout vkuiFixedLayout--vertical-top vkuiPanelHeader__fixed vkuiInternalPanelHeader__fixed"
    data-onboarding-tooltip-container="fixed"
    style={
      {
        "width": undefined,
      }
    }
  >
    <div
      className="vkuiPanelHeader__in"
      data-onboarding-tooltip-container="fixed"
    >
      <div
        className="vkuiPanelHeader__before vkuiInternalPanelHeader__before"
      />
      <div
        className="vkuiPanelHeader__content"
      >
        <span
          className="vkuiPanelHeader__content-in"
        >
          Hello
        </span>
      </div>
      <div
        className="vkuiPanelHeader__after vkuiInternalPanelHeader__after"
      />
    </div>
  </div>
  <div
    className="vkuiSeparator vkuiSeparator--padded vkui-sizeX--compact-mq"
  >
    <hr
      className="vkuiSeparator__in"
    />
  </div>
  <div
    className="vkuiSpacing vkui-sizeX--regular-mq"
    style={
      {
        "height": 16,
        "padding": "8px 0",
      }
    }
  />
</div>
`;
